templates/dashboard/alias_contact_manager.html

```diff
@@ -133,6 +133,7 @@ <h1 class="h3">
           <div>
             <span>
               <a href="{{ 'mailto:' + contact.website_send_to() }}"
+                 target="_blank"
```

Comment:
Hey I'm not sure what is the benefit of opening this link in a new tab?

Reply:
That's especially for this link, that I created this MR 😅.

When I open a new application, I like having it in a new tab, allowing me to switch from one to another without losing the change I did on one. It is also easier if I navigate a lot in my email client and want to go back to SL, I don't have to press the "previous" button X times.

But it is a personal preference

Reply:
I haven't tried on other OSes but in my case, the browser will ask me to open an email application without navigating to another page so I don't need to use the back button. I'm wondering if this only happens if the default email client is set to a web mail ?

Reply:
Yes, I was talking about having the default email client set to a web mail. I tried also with the default mail client set to Thunderbird and opening the "mailto" links into a new tab, and it has no effect when the mail client is a desktop app.